assignments/002-unix-intro/unix-intro.sh

```diff
@@ -0,0 +1,1085 @@
+(base) [c836820202@colostate.edu@login12 002-unix-intro]$
+
+#### Ex. 18: Using globbing, remove `dir_A`, `dir_C`, `dir_E`, `dir_G`, and `dir_I`,
+####         using a command with 8 characters (including spaces)
+
+rm d* -r
+
+
+#### Ex. 19: Create a directory called `nice_dir_name` and then use `echo`,
+####         redirecting its output to create a file called `nice_file_name`
+####         inside of `nice_dir_name` whose contents is the string
+####         `A string in a file!`
+
+(base) [c836820202@colostate.edu@login12 002-unix-intro]$ mkdir nice_dir_name
+
+(base) [c836820202@colostate.edu@login12 002-unix-intro]$ echo 'A string in a file!' > nice_dir_name/nice_file_name
+
+
+#### Ex. 20: Catenate the contents of `nice_dir_name/nice_file_name` to _stdout_
+
+(base) [c836820202@colostate.edu@login12 002-unix-intro]$ cat nice_dir_name/nice_file_name
+A string in a file!
+
+
+#### Ex. 21: Create a directory called `bad directory name with spaces`, and then,
+####         with redirection, create a file inside it, named 
+####         `bad file name with spaces` whose contents are the string
+####         `Whoa! No spaces, please!`. Use backslash escaping instead of quoting
+
+(base) [c836820202@colostate.edu@login12 002-unix-intro]$ mkdir bad\ directory\ name\ with\ spaces
+(base) [c836820202@colostate.edu@login12 002-unix-intro]$ echo 'Whoa! No spaces, please!' > bad\ directory\ name\ with\ spaces/bad\ file\ name\ with\ spaces
+
+
+#### Ex. 22: Catenate the contents of 
+####         `bad directory name with spaces/bad file name with spaces` to stdout.
+
+(base) [c836820202@colostate.edu@login12 002-unix-intro]$ cat bad\ directory\ name\ with\ spaces/bad\ file\ name\ with\ spaces
+Whoa! No spaces, please!
+
+
+#### Ex. 23: List the directories (just the names, not the contents (`-d`)) of
+####         `bad directory name with spaces` and `nice_dir_name`.
+
+(base) [c836820202@colostate.edu@login12 002-unix-intro]$ ls bad\ directory\ name\ with\ spaces/ nice_dir_name/ -d
+bad directory name with spaces/  nice_dir_name/ -d
+
+
+
+#### Ex. 24: Once you are convinced that you can list the directories
+####         correctly in your terminal, recall that `ls` command (i.e. get it back on the
+####         command line using the up arrow on your keyboard) and replace `ls` with
+####         `rm -r` to remove them. BE CAREFUL with the rm command!
+
+ rm bad\ directory\ name\ with\ spaces/ nice_dir_name/ -r
+
+
+#### Ex. 25: Why is it a good idea to `ls` things before removing them?
+
+If you 'ls' prior to removal you can verify the contents of a directory so that you do not remove something you did not want to
+
+#### Ex. 26: If you had just typed `rm -r bad directory name with spaces` 
+####         what would Unix have tried to remove? (No command to type here,
+####         just answer the question).
+
+Unix would have removed the working directory and would not have recognized 'bad directory name with spaces'
```

Comment:
Not quite.  It would try to remove directories named:

- bad
- directory 
- name
- with 
- spaces